youtube/utils.py

```diff
@@ -7,7 +7,7 @@
 def convert_tmp_video(src_path, dst_path):
     stdout = None
     stderr = None
-    """mffmpeg -i input-file.ogv -strict experimental -pix_fmt yuv420p -r 15 -f mp4 tmp.mp4"""
+    """mffmpeg -i input-file.webm -strict experimental -pix_fmt yuv420p -r 15 -f mp4 tmp.mp4"""
     print(("/usr/bin/mffmpeg -i", src_path, "-strict experimental -pix_fmt yuv420p -r 15 -f mp4", dst_path))
```

Comment:
Is this code in use? Is mffmpeg different from ffmpeg? I can't find the mffmpeg command on my system.

If ffmpeg is the correct name, change mffmpeg to ffmpeg everywhere.